packages/type/scss/_reset.scss

```diff
@@ -25,19 +26,53 @@
 
   body {
     font-family: $body-font-family;
-    font-weight: 400;
+    @include carbon--font-weight('regular');
     text-rendering: optimizeLegibility;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
   }
 
-  // IBM Plex uses semibold instead of bold, as a result we need to map
-  // tags that use `font-weight: bold` to the semibold value
-  strong {
-    font-weight: 600;
-  }
-
   code {
     font-family: $mono-font-family;
   }
+
+  h1 {
+    @include carbon--type-style('productive-heading-06');
```

Comment:
https://deploy-preview-3648--the-carbon-components.netlify.com/?nav=type

Why is the h1 here 32px? Shouldn't it be 42px according to `productive-heading-06` here? https://www.carbondesignsystem.com/guidelines/typography/productive

Reply:
I mean, I'm using the token for that. When I inspect it you can see the styles it's pulling are directly from the token. If there's an issue with the font size that's an issue with the token @mattrosno The type token has `font-size: carbon--type-scale(8)` which I believe is `32px`

Reply:
I see productive-heading-06 is also 32px here. https://carbon-elements.netlify.com/type/examples/preview/

Can you open a bug to find out if our documentation is wrong for productive-heading-06? Or if there's a bug in `@carbon/types`?